test/IntegrationTests/SmokeTests.cs

```diff
@@ -145,6 +149,39 @@ public void SubmitMetrics()
         }
     }
 
+    [Fact]
+    [Trait("Category", "EndToEnd")]
+    public async Task PrometheusExporter()
+    {
+        SetEnvironmentVariable("LONG_RUNNING", "true");
+        SetEnvironmentVariable("OTEL_METRICS_EXPORTER", "prometheus");
+        SetEnvironmentVariable("OTEL_DOTNET_AUTO_METRICS_ADDITIONAL_SOURCES", "MyCompany.MyProduct.MyLibrary");
+        string defaultPrometheusMetricsEndpoint = "http://localhost:9464/metrics";
```

Comment:
In the future, we could use `OTEL_EXPORTER_PROMETHEUS_PORT` to avoid a port collision, however right now this configuration option is not available.

Reference: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/852